ledger/src/shred.rs

```diff
@@ -947,6 +917,7 @@ pub fn max_entries_per_n_shred(
     (shred_data_size * num_shreds - count_size) / entry_size
 }
 
+#[cfg(feature = "dev-context-only-utils")]
```

Comment:
Generally, this is a good change but this doesn't fall under the category of what the commit title is saying